src/styles/block/main.scss

```diff
@@ -0,0 +1,37 @@
+body {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  align-items: center;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  margin-left: 4px;
+  display: flex;
+  flex-wrap: wrap;
+  padding-inline: 10px;
+  gap: $gapValue;
```

Comment:
It seems that you are using a variable `$gapValue` here. Make sure that this variable is defined in your SCSS files, otherwise it will cause an error.